src/fides/api/custom_types.py

```diff
@@ -66,8 +66,13 @@ def validate_html_str(val: str) -> str:
         "strong",
         "u",
     }
+
+    ALLOWED_ATTRIBUTES = {
+        "a": {"href", "target"},
```

Comment:
Hi @Vagoasdf, the allowed attributes by default ammonia are `href` and `hreflang`, so I think we should add `hreflang` to this list.

https://docs.rs/ammonia/latest/ammonia/struct.Builder.html#defaults-2